rosbag2_storage_default_plugins/src/rosbag2_storage_default_plugins/sqlite/sqlite_storage.cpp

```diff
@@ -278,7 +265,11 @@ rosbag2_storage::BagMetadata SqliteStorage::get_metadata()
   metadata.starting_time =
     std::chrono::time_point<std::chrono::high_resolution_clock>(std::chrono::nanoseconds(min_time));
   metadata.duration = std::chrono::nanoseconds(max_time) - std::chrono::nanoseconds(min_time);
-  metadata.bag_size = rosbag2_storage::FilesystemHelper::calculate_directory_size(database_name_);
+  metadata.bag_size = 0;
+
+  for (const auto & relative_path : metadata.relative_file_paths) {
```

Comment:
I am confused. How can `metadata.relative_file_paths` ever contains multiple files in the current implementation?

Reply:
This change was done because `calculate_directory_size` was determined to not be the best way to calculate `bag_size` in another issue.

Since `metadata.relative_file_paths` is hardcoded now to `get_relative_file_paths()`, I am replacing this accumulating for-loop with `calculate_directory_size(get_relative_file_path())`